src/routes/news/containers/TunewsDetailsPage.js

```diff
@@ -55,15 +55,18 @@ type PropsType = {|
   tunewsElement: TunewsModel,
   language: string,
   path: string,
-  city: string
+  city: string,
+  cities: Array<CityModel>
 |}
 
 export class TunewsDetailsPage extends React.PureComponent<PropsType> {
   render () {
-    const { tunewsElement, language, path, city } = this.props
+    const { tunewsElement, language, path, city, cities } = this.props
 
-    if (!tunewsElement) {
-      const error = new ContentNotFoundError({ type: 'tunewsItem', id: path, city, language })
+    const currentCity: CityModel = cities && cities.find(cityElement => cityElement.code === city)
+    if (!tunewsElement || !currentCity.tunewsEnabled) {
+      const type: string = currentCity.tunewsEnabled ? 'tunewsItem' : 'category'
```

Comment:
In the LocalNewsPage.js we always choose 'category'. Which one should we use here?

Reply:
If the tu-news feature is enabled, I'd show an error that the corresponding tu-news-Element does not exist.
If it's disabled, this route is invalid, so imo a generic CategoryNotFoundError should be shown.

Reply:
This will be reworked in WEBAPP-569 anyways. So it's not that important I think.

Reply:
@maxammann this is the detail page, so it is correctly done imo and has the same behavior as the `LocalNewsDetailsPage`

Reply:
@NKlug why do you think this will be reworked in `WEBAPP-569`?

Reply:
I guess a didn't read the whole thing properly. I thought it involved redirecting to the TunewsPage if the TunewsElement has not been found/does not exist.